content/F2015/F15-event-dropbox-coffee.md

```diff
@@ -0,0 +1,16 @@
+title: Dropbox Coffee Chats
+Date: 2015-09-05
+Category: Events
+Tags: social, industry
+Slug:  dropbox-coffee-chats
+Author: Anna Lorimer
+Summary: Come and grab some coffee and snacks and chat with some female engineers from Dropbox!
+
+Join us and several female engineers from Dropbox for coffee and snacks at William's in the plaza.
+
+## Event Details ##
+
++ **Who:** This is a [women-only](http://wics.uwaterloo.ca/faq/) event.
```

Comment:
How about [women-only]({filename}/pages/faq.md) ? Just to make the links relative 😺